plugins/audio-buffer.hh

```diff
@@ -39,6 +39,7 @@ namespace clap {
          // So take a copy and keep the _dataBase pointer around.
          auto tmp = _dataBase;
          _data = static_cast<T*>(std::align(alignment, dataSize, tmp, dataSizeLeft));
+         setConstant(false);
```

Comment:
Why?
A better fix would be to make initialize the audio buffer to a constant value of zero.

Reply:
This is meant to set `_stride = _channelCount`. Otherwise _stride is 0 and _channelCount is 2

Reply:
after initialization, the audio buffer should have a constant value of 0

Reply:
At some point someone has to set _stride = _channelCount, otherwise you copy only from the left channel. This is what I saw at runtime in a debugger inside toClap